editor/src/components/shared/scoped-variables.ts

```diff
@@ -141,7 +141,7 @@ function getContainerPropsValue(
       if (isJSXAttributesEntry(prop)) {
         const value = simplifyAttributeIfPossible(prop.value)
         if (isJSXAttributeValue(value)) {
-          runtimeProps[prop.key] = value.value
+          runtimeProps[prop.key] = { spiedValue: value.value as unknown }
```

Comment:
originally value.value is `any` here which defeated the type checker, so I put this `as unknown` here as hardening.. maybe it's just lipstick on a pig though

Reply:
it's definitely better than `any`, thanks for this fix